gio/src/auto/file.rs

```diff
@@ -251,6 +251,26 @@ pub trait FileExt: 'static {
         cancellable: Option<&P>,
     ) -> Result<FileEnumerator, glib::Error>;
 
+    #[doc(alias = "g_file_enumerate_children_async")]
```

Comment:
Oh, you'd add this to `src/file.rs` into the `FileExtManual` trait.

Reply:
oops, will fix